core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerProcess.java

```diff
@@ -110,7 +110,7 @@
       // registered at worker registry, so the maximum timeout here is set to the multiply of
       // the number of factories by the default timeout of getting a worker from the registry.
       CommonUtils.invokeAll(callables,
-          (long) callables.size() * 10 * Constants.DEFAULT_REGISTRY_GET_TIMEOUT_MS);
+          (long) callables.size() * ServerConfiguration.getMs(PropertyKey.WORKER_STARTUP_TIMEOUT));
```

Comment:
if we're using a configurable timeout, do we still need the multiply of the number of factories?

Reply:
No, we don't need anymore.